### DISCUSSION TITLE
Why are the Jailbreak guards stronger than Endworms?

### DISCUSSION THREAD

**Discussion by local-interloper** [FakeFishGames/Barotrauma#13125] [Ideas and suggestions]
A single one can wipe out a squad of 2-3 without any problem. I just emptied 1/3 of an HMG into one and they didn't even flinch. This doesn't see particularly balanced.

**Comment by SomeRandomNoobKekeke** [comment]
It's imposible to complete jailbreak or just wipe hostile outpost with early game weapons
But if you infect the guards their husks seems to inherit health multiplier and become unstoppable for them
Also why traiders have x3 health multiplier? For fake poison resistance? Ok, i'll just use 2 syringes

**Comment by Regalis11** [comment]
Outpost security was made this tough because otherwise it would've been extremely easy to wipe out entire outposts. Same with the traders, without the health multiplier it'd be very easy for e.g. a griefer to kill them and prevent the crew from trading.

I do agree though that it feels really odd and immersion-breaking, so I think we could consider alternative ways to make them tougher. But in any case, I don't think this is a bug report, so I'll move this to discussions.

>It's imposible to complete jailbreak or just wipe hostile outpost with early game weapons

I disagree with this. Killing the security is not the only way to complete a jailbreak mission, you can also just avoid being seen by them, distract them somehow or incapacitate them. I also think that it should not be next to impossible to wipe out a hostile outpost with early game weapons, and even with more advanced weapons, it should be very difficult.

**Reply by local-interloper** [reply]
They are literally omnipotent in regards to station damage (this can could be exploted as a distraction but it really feels immersion breaking as you mentioned), if you're spotted they zero on you in a matter of seconds, their guns stunlock you like crazy. Incapacitating them is extremely difficult as they resist most pain shock and stun granades are a bit janky to throw. This was one of the techniques we used tho. We would stun them and then cuff them. The game's stealth systems are not advanced enough to make the sneaking approach fun. Our current technique is to enable destruction of stations and flood the station itself till nothing but the prisoners is alive. Then drain the station and escort the prisoners. If this is a preferable alternative I would say there's some space for improvement. Speaking of improvement, I have some ideas:

- Have the players dress up as employees of the station and force em to blend in
- Improve the stealth capabilities
- Make em less tough but spawn them continuously in waves
- Abandoned outpost missions are genuinely fun due to their balanced difficulty, perhaps turning prisoner rescue into a outpost clear mission with an extra task wouldn't be too bad of a change?

**Reply by SomeRandomNoobKekeke** [reply]
I prefer to drill prisoners cell from outside, give them suits and escape by water

I also have an idea how to fix this: 
- make more security guard variations and spawn them based on level difficulty instead of commonness

**Reply by neirenoir** [reply]
Maybe overkill, but what about spawning/undocking attack shuttles to siege the player sub in case they cause too much ruckus?

**Comment by BlackMoons** [comment]
>otherwise it would've been extremely easy to wipe out entire outposts.

My submarines electrical system can wipe outposts at the flick of a switch... Given a few minutes. PS: Fires on indestructible outposts are nearly impossible to put out and AI's all die trying to put them out.

> I prefer to drill prisoners cell from outside, give them suits and escape by water

Can you open their cells, give them some diving masks/O2 tanks and close the cell doors so other AI's can't steal the tanks?

**Reply by SomeRandomNoobKekeke** [reply]
I think if you open the door the security will shred you
![barodev small](https://github.com/Regalis11/Barotrauma/assets/122838333/c280d35b-a414-49ea-9ba4-23640333b6fa)

**Comment by SomeRandomNoobKekeke** [comment]
It took me a little longer than i might expect, but [here it is](https://steamcommunity.com/sharedfiles/filedetails/?id=3110897421)
